build/package/generate-msi.sh

```diff
@@ -50,3 +50,9 @@ env GOOS=windows GOARCH=amd64 go build \
 go-msi make --path "${WIX_MANIFEST_FILE}"  --msi "dist/${PACKAGE_NAME}" --version "${VERSION}"
 
 go run ./tools/sign -file "dist/${PACKAGE_NAME}"
+
+if [[ "${TOOL_NAME:?}" == atlascli ]]; then
+  go run ./tools/update-chocolatey-package/update-chocolatey-package.go "${VERSION}" ./dist/"${PACKAGE_NAME}" https://fastdl.mongodb.org/mongocli/"${PACKAGE_NAME}"
+  cd ./build/package/chocolatey/temp
+  choco pack
```

Comment:
nit: I'd probably have called it from go

Reply:
Done!